decidim-meetings/spec/commands/create_meeting_spec.rb

```diff
@@ -168,7 +168,25 @@ module Decidim::Meetings
 
         expect(UpcomingMeetingNotificationJob)
           .to receive_message_chain(:set, :perform_later) # rubocop:disable RSpec/MessageChain
-          .with(set: start_time - 2.days).with(1, "1234")
+          .with(set: start_time - Decidim::Meetings.upcoming_meeting_notification).with(1, "1234")
+
+        allow(Decidim::EventsManager).to receive(:publish).and_return(true)
+
+        subject.call
+      end
+
+      it "doesn't schedule an upcoming meeting notification if start time is in the past" do
```

Comment:
Any reason why this spec doesn't follow the same pattern as the others? The behavior should be the same always, right?

Reply:
This is a stubbed object as we are simulating the form input. 
In this particular case, we cannot apply the pattern applied in the other files.